stdlib/2and3/keyword.pyi

```diff
@@ -0,0 +1,6 @@
+# Stubs for keyword
+
+from typing import Sequence
+
+def iskeyword(s: str) -> bool: ...
```

Comment:
Correct, it works on every `Hashable`, but I'm unable to find a concret example where it would be correct to test that a `frozenset` is a keyword, it would be better to trigger a warning.

In CPython implementation, the code is simply `iskeyword = frozenset(kwlist).__contains__`, I don't think that was intended to handle testing other type of value, it works, but that's not the point.